src/scripts/convertSvg.ts

```diff
@@ -65,15 +68,22 @@ export const convertSvg = async (toolbox: GluegunToolboxExtended, cleanup = fals
     await cmd(`rm -rf ${SVGR_TEMPLATE_DIR}`);
     await cmd(`rm -rf _temp`);
     await cmd(`rm -rf ${FILENAME_SVGO_CONFIG}`);
+    // remove generated index.tsx file
+    await cmd(`rm -rf ${path(OUT_DIR, 'index.tsx')}`);
```

Comment:
Personally I try to stay away from indexing component files, as I prefer to import the specific files to avoid unnecessary imports.